frontend/src/store/modules/sqlEditor.ts

```diff
@@ -135,6 +112,12 @@ const getters = {
     const connectionContext = state.connectionContext;
     return `${connectionContext.instanceId}/${connectionContext.databaseId}/${connectionContext.tableId}`;
   },
+  isDisconnected(state: SqlEditorState) {
+    return (
+      state.connectionContext.instanceId === 0 ||
+      state.connectionContext.databaseId === 0
```

Comment:
I am not sure if this is correct, for mysql, it's not required to connect a particular database